src/Lean/Meta/AppBuilder.lean

```diff
@@ -102,6 +102,7 @@ def mkEqTrans (h₁ h₂ : Expr) : MetaM Expr := do
     | none, _ => throwAppBuilderException ``Eq.trans ("equality proof expected" ++ hasTypeMsg h₁ hType₁)
     | _, none => throwAppBuilderException ``Eq.trans ("equality proof expected" ++ hasTypeMsg h₂ hType₂)
 
+/-- Given `HEq a b`, makes `HEq b a`.  -/
```

Comment:
```suggestion
/-- Given `h : HEq a b`, return a proof of `HEq b a`.  -/
```
And similarly for the other ones, to distinguish the proving functions from ones which would manipulate types.

Reply:
Should it be `$h : HEq a b`?

Reply:
I don't think we use `$` in this sense in docstrings elsewhere